ansible/accumulo.yml

```
@@ -53,26 +53,32 @@
       when: accumulo_major_version == '2' and not use_systemd
 - hosts: accumulomaster
   tasks:
-    - name: "start accumulo 2.0 master, monitor, gc & tracer"
+    - name: "start accumulo 2.x master, monitor, and gc"
       command: "nohup {{ accumulo_home }}/bin/accumulo-service {{ item }} start"
       register: start_result
       changed_when: "'Starting' in start_result.stdout"
       with_items:
         - master
         - monitor
         - gc
-        - tracer
       when: accumulo_major_version == '2' and not use_systemd
+    - name: "start accumulo 2.0 tracer"
+      command: "nohup {{ accumulo_home }}/bin/accumulo-service tracer start"
+      when: accumulo_version is version('2.0.0','>=') and accumulo_version is version('2.1.0','<') and not use_systemd
```

To be equivalent to what's in the other task, I think you also need:

```suggestion
      when: accumulo_version is version('2.0.0','>=') and accumulo_version is version('2.1.0','<') and not use_systemd
      register: start_tracer_result
      changed_when: "'Starting' in start_tracer_result.stdout"
```

---

yes, what this would do is to make Ansible report a green "ok" status for that play / task, when tracer is already running.